src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -103,6 +103,8 @@ def __call__(
         latents: Optional[torch.FloatTensor] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        callback: Optional[Callable] = None,
```

Comment:
Specify the signature of the callback:
```suggestion
        callback: Optional[Callable[[int, List[PIL.Image.Image]], None]] = None,
```

(and same goes for where this appears in the other pipelines)